docs/reference/commandline/dockerd.md

```diff
@@ -72,6 +72,7 @@ Options:
       --max-concurrent-uploads int            Set the max concurrent uploads for each push (default 5)
       --metrics-addr string                   Set default address and port to serve the metrics api on
       --mtu int                               Set the containers network MTU
+      --node-generic-resources list           Advertise user-defined resource
```

Comment:
I guess the docs need to be updated, the flag should be  `--generic-resource` not `--node-generic-resources`

Reply:
That flag is for the docker daemon in this case it is ` --node-generic-resources`

Reply:
ah I missed that, sounds good